src/app/pages/FrontPage/index.jsx

```diff
@@ -30,6 +30,11 @@ import LinkedData from '#containers/LinkedData';
 import ATIAnalytics from '#containers/ATIAnalytics';
 import ChartbeatAnalytics from '#containers/ChartbeatAnalytics';
 
+export const StyledFrontPageMain = styled.main`
+  background-color: ${C_GHOST};
+  z-index: 0;
```

Comment:
The bars on section label components have a `z-index` of -1, we're defining this `z-index` so that the bars are rendered on top of this `main`.

Reply:
This will be addressed in https://github.com/bbc/psammead/pull/3195